person.rb

```diff
@@ -20,6 +21,10 @@ def correct_name
     @name
```

Comment:
- [ ] It would be better if we initialize our rental variable. For more information about it, kindly read this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars)